README.md

```diff
@@ -168,6 +170,43 @@ python3 ecosify-strings.py Extensions
 python3 ecosify-strings.py Shared
 ```
 
-## Release 🚀
+## :rocket: Release
 
 Follow the instructions from our [confluence page](https://ecosia.atlassian.net/wiki/spaces/MOB/pages/2460680288/How+to+release)
+
+## Howto update the release notes (semi-automated)
+
+Make sure that `fastlane` and `transifex`-cli is installed.
+
+### Add source release notes to transifex (English)
```

Comment:
Is it needed to specify American English here as we have `en-US` ?

Reply:
yeah good to be precise here 👍